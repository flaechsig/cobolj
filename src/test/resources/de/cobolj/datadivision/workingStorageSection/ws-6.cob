IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-ID   PIC S9(7) VALUE '-12345'.

PROCEDURE DIVISION.
   DISPLAY "My ID is : " WS-ID.
STOP RUN.