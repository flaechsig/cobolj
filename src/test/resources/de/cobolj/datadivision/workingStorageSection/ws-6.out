My ID is : -0012345
